.circleci/config.yml

```diff
@@ -47,10 +47,11 @@ jobs:
           command: kubectl create -f /cncf/rbac/
           environment:
             KUBECONFIG: /cncf/data/kubeconfig
-      - persist_to_workspace:
-          root: /cncf/data
+      - run: find /cncf/data
+      - save_cache:
+          key: cncf-data-{{.Environment.CIRCLE_WORKFLOW_ID}}
           paths:
-            - ./*
+            - data
```

Comment:
For my own understanding, is this saving `cncf-data-{{.Environment.CIRCLE_WORKFLOW_ID}}` in the directory `data`? If so, should we put an absolute path here rather than what looks to be a relative path?

Reply:
Not quite ;)  For reference, the [CircleCI Caching docs](https://circleci.com/docs/2.0/caching/) are pretty good, but here's the tl;dr version :)
```cncf-data-{{.Environment.CIRCLE_WORKFLOW_ID}}``` is the cache key.  You can write to a cache key precisely *once*.  Its write once, and then read only.

The ```{{.Environment.CIRCLE_WORKFLOW_ID}}``` expands to the environment variable CIRCLE_WORKFLOW_ID, which has the id for this workflow run.  paths tells us which directories to put into the 'cache' for that cache key.  So it goes and grabs everything in directory (relative to the working dir, which is /cncf/ and puts it in the cache under the cache key cncf-data-{{.Environment.CIRCLE_WORKFLOW_ID}}

Then when we use the CircleCI API via curl later we set the CIRCLE_WORKFLOW_ID, and it will look up the correct cache we put aside here.  The /cncf/data dir has all the info needed to clean up in packet-destroy.

Make sense?